release.sh

```diff
@@ -81,9 +81,21 @@ git push origin "$TAG_NAME"
 
 echo "**** Pushing build $BUILD ****"
 
+rm -f $TARBALL.sig install.sh.sig
+
+# Sign the tarball and the install script. Note that we don't sign the channel build number because
+# it wouldn't accomplish anything: If an attacker wanted to provide an old number, they could
+# provide the old signature to match. If an attacker provided a number that hasn't been used
+# before, they would not be able to provide a matching package because no such signed package
+# exists.
+gpg -u 9DA8CCBD --detach-sig $TARBALL
```

Comment:
Consider specifying the full GPG fingerprint here. I _think_ there's no workable attack if you take care of development workstation's keyring, but I figure, better safe than sorry.